vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Install required Ubuntu packages
+  block:
+    - name: Update apt
+      apt: update_cache=yes name=docker-ce state=latest
+
+    - name: Fetch yarn gpg key
+      apt_key:
+        url: https://dl.yarnpkg.com/debian/pubkey.gpg
+        state: present
+
+    - name: Add yarn release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/yarn.list
+        line: deb https://dl.yarnpkg.com/debian/ stable main
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Add OS release key
+      lineinfile:
+        path: /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+        line: "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/ /"
+        create: yes
+        owner: root
+        group: root
+        mode: '0644'
+
+    - name: Fetch OS release key
+      apt_key:
+        url: "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_{{ OS_VERSION_ID }}/Release.key"
+        state: present
+
+    - name: Update apt
+      apt: update_cache=yes name=docker-ce state=latest
+
+    - name: Set update-alternatives to python3
+      community.general.alternatives:
+        name: python
+        path: /usr/bin/python3.6
+        link: /usr/bin/python
+      when: OS_VERSION_ID == "18.04"
+
+    - name: Set update-alternatives to python3
+      community.general.alternatives:
+        name: python
+        path: /usr/bin/python3.8
+        link: /usr/bin/python
+      when: OS_VERSION_ID != "18.04"
+
+    - name: Podman
+      block:
+        - name: Install podman
+          apt: name=podman
+          
+        - name: Default registries configuration file
+          blockinfile:
+            path: /etc/containers/registries.conf
+            block: |
+              [registries.search]
+              registries = ['docker.io', 'quay.io']
```

Comment:
Could the right side of the `=` be replaced by env var like,  `${REGISTRY}` in the original .sh file?

Reply:
Yes of course. But interestingly that's why I got after running make in a environment with container runtime set to podman. I'm not sure if it is the issue with the script that it is not setting REGISTRY in the list properly or something else. I could for example do like this:
```
name: Template registries configuration             
template: 
        src: "{{ DAEMON_JSON_PATH }}/registries.json"
        dest: /etc/containers/registries.conf
        owner: root
        group: root
# {{ DAEMON_JSON_PATH }}/registries.json before the templating

cat registries.json
 [registries.search]
 registries = ['docker.io', 'quay.io', "{{ REGISTRY }}"]
```

Reply:
and then when we run template task, the output would be

```
 [registries.search]
 registries = ['docker.io', 'quay.io', '192.168.111.1:5000']
```

Reply:
Fixed